cypress/integration/pages/articles.js

```diff
@@ -15,48 +15,27 @@ const serviceHasCorrectlyRenderedParagraphs = service => service !== 'sinhala';
 const runTests = ({ service }) =>
   describe(`Tests`, () => {
     describe(`Metadata`, () => {
-      it('should have lang and dir attributes', () => {
-        cy.request(`${config[service].pageTypes.articles.path}.json`).then(
-          ({ body }) => {
-            cy.hasHtmlLangDirAttributes({
-              lang: body.metadata.passport.language,
-              dir: appConfig[service].dir,
-            });
-          },
-        );
-      });
-
-      it('should have the correct facebook metadata', () => {
-        cy.checkFacebookMetadata(
-          '100004154058350',
-          '1609039196070050',
-          `${appConfig[service].articleAuthor}`,
-        );
-      });
-
-      it('should have the correct open graph metadata', () => {
-        cy.checkOpenGraphMetadata(
-          'Meghan follows the royal bridal tradition started by the Queen Mother in 1923.',
-          `${appConfig[service].defaultImage}`,
-          `${appConfig[service].defaultImageAltText}`,
-          `${appConfig[service].locale}`,
-          `${appConfig[service].defaultImageAltText}`,
-          "Meghan's bouquet laid on tomb of unknown warrior",
-          'article',
-          `https://www.bbc.com${config[service].pageTypes.articles.path}`,
-        );
-      });
-
-      it('should have the correct twitter metadata', () => {
-        cy.checkTwitterMetadata(
-          'summary_large_image',
-          `${appConfig[service].twitterCreator}`,
-          'Meghan follows the royal bridal tradition started by the Queen Mother in 1923.',
-          `${appConfig[service].defaultImageAltText}`,
-          `${appConfig[service].defaultImage}`,
-          `${appConfig[service].twitterSite}`,
-          "Meghan's bouquet laid on tomb of unknown warrior",
-        );
+      it('should have the correct articles metadata', () => {
+        cy.checkArticlesMetadata({
```

Comment:
only used once shouldn't be a command

Reply:
I will factor the values out of `checkArticlesMetadata` and into `checkSharedSocialmediaMetadata` in a future PR following this one, as mentioned in this comment: https://github.com/bbc/simorgh/pull/3333/files#diff-0699cce6bef2847b0f13e4bd5f6394c2R7. It is clearer to leave in the commands in `metaData.js` for the moment.

Reply:
Ok pls add an issue